app/lib/server/lib/debug.js

```diff
@@ -79,7 +81,7 @@ const originalMeteorPublish = Meteor.publish;
 Meteor.publish = function(name, func) {
 	return originalMeteorPublish(name, function(...args) {
 		traceConnection(Log_Trace_Subscriptions, Log_Trace_Subscriptions_Filter, 'subscription', name, this.connection, this.userId);
-		logger.publish(name, '-> userId:', this.userId, ', arguments: ', args);
+		logger.publish(name, '-> userId:', this.userId, ', arguments: ', omitKeyArgs(args));
```

Comment:
to not waste the processing time used to omit fields and not even show the log (because of the log level) I'd consider using the deferred version that is called only when the log will be shown:
```suggestion
		logger.publish(() => `${ name } -> userId: ${ this.userId }, arguments: ${ omitKeyArgs(args) }`);
```